src/background/Wallet/WalletRecord.ts

```diff
@@ -772,6 +771,7 @@ export class WalletRecordModel {
       upgradeDnaBannerDismissed: false,
       backupReminderDismissedTime: 0,
       enableTestnets: false,
+      testnetMode: null,
```

Comment:
What about moving this to globalPreferences? This param can be easier to read from globally available class

Reply:
Why? Unless the value should be read when you're logged out, it doesn't belong to globalPreferences store

Reply:
Ok